addon/components/o-s-s/country-selector.ts

```diff
@@ -54,6 +58,19 @@ export default class OSSCountrySelector extends Component<OSSCountrySelectorArgs
       : this.intl.t('oss-components.country-selector.placeholder.province');
   }
 
+  @action
+  handleKeyEvent(e: KeyboardEvent): void {
+    if (!this.dropdownVisibility) {
```

Comment:
I don't get the if/else usages here nor the check for visibility, can't something like this work ?
```typescript
if (e.key === 'Enter' || e.key === 'Escape' || e.key === 'Tab') {
   this.toggleDropdown(e);
}
```

Reply:
if you do need to check for visibility for the "Enter" key, you can at least have:
```
if (!this.dropdownVisiblility && e.key === 'Enter') this.toggleDropdown(e)
else if (this.dropdownVisibility && (e.key === 'Escape' || e.key ===' Tab') this.toggleDrodpown(e)
```

Reply:
Simplify the behavior in `country-selector` & improve the close behavior in `infinite-select`